tomviz/python/Recon_TV_minimization.py

```diff
@@ -87,11 +80,6 @@ def transform_scalars(self, dataset, Niter=10, Nupdates=0):
                     f = f + row * a * beta
                 recon[s, :, :] = f.reshape((Nray, Nray))
 
-                # Give 4 updates for first iteration.
-                if Nupdates != 0 and i == 0 and (s + 1) % (Nslice//4) == 0:
-                    utils.set_array(child, recon)
-                    self.progress.data = child
-
```

Comment:
Are these statements not needed?

Reply:
It was to provide an estimate of the object for the first estimate, but we also provide live updates on lines 95-98.